public/index.html

```diff
@@ -2,14 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/logo.png" />
+    <link rel="icon" href="%PUBLIC_URL%/newLogo.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="We are Azerbaijan based innovative design & development studio that offers web based products. We aim to deliver the best results, therefore, we approach every project with special care and we pay attention to every single detail so that we deliver what we promised"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo.png" />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/newLogo.png" />
```

Comment:
that's nice that you added `newLogo.png`, but you still forgot to add 
favicon for non-apple users:
please read the below article (example part): 
https://www.w3schools.com/html/html_favicon.asp